app/src/main/res/values-fr/strings.xml

```diff
@@ -143,4 +143,8 @@
     <string name = "biking_newcomer">Novice du cyclisme</string>
     <string name ="biking_amateur">Amateur de cyclisme</string>
     <string name ="biking_expert">Expert en cyclisme</string>
+    <string name="login_swipe_text">Glisse pour plus d\'infos…</string>
+    <string name="login_activities_text">Lasta vous permet de trouver des activités de plein air à proximité et de suivre vos performances.</string>
+    <string name="login_friends_text">Vous êtes plutôt un sportif social ? Nous avons ce qu\'il vous faut ! Invitez et ajoutez vos amis dans l\'application et regardez leurs dernières activités.</string>
+    <string name="login_free_text">Le plus beau ? C\'est totalement gratuit ! Essayez-le maintenant avec une simple connexion Google et rejoignez notre communauté de fans de sport.</string>
```

Comment:
"fans de sports" is more of someone that WATCHES professional sports rather than does sport. I'd write "sportifs" instead, but it's debatable.

Reply:
I changed it due to Andrew's request, is it ok for you now ?
"" Le plus beau ? C\'est totalement gratuit ! Inscrivez-vous pour rejoindre notre fantastique communauté d\'accros au sport ! ""

Reply:
Yessir, perfect